domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
 smart-co.co.uk
 smartedgesolutions.co.uk
 smartsourcing.plc.uk
 snap.hr
+socode.co.uk
 softwarepersonnel.co.uk
 solagroup.com
 soprasteria.com
+soultek.co.uk
 sourced.ai
 sourced.tech
 southern-lights.co.uk
 spectrumit.co.uk
 speerhead.com
 spencer-rose.com
 spgsearch.com
+spherelondon.co.uk
 spring.com
+sqasearch.co.uk
 sqcp.com
 squareone.eu
 squareoneresources.com
 squareoneresourcesnews.com
+staffhost.co.uk
 staffingprime.com
 staffline.co.uk
+staffworx.co.uk
```

Comment:
We are a specialist agency serving the eCommerce, software, consulting and digital sectors.

Staffworx has built a reputation for excellence in delivery based on over 20 years of recruitment in the online retail, systems integration, software vendor and business consulting sectors.